apps/showcase/src/assets/trainings/sdk/steps/angular-integration/exercise/app.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, inject, signal } from '@angular/core';
+import { ApiFactoryService } from '@o3r/apis-manager';
+import { type Pet, PetApi, StoreApi } from 'sdk';
+
+@Component({
+  selector: 'app-root',
+  standalone: true,
+  imports: [],
+  templateUrl: './app.component.html',
+  styleUrl: './app.component.scss'
+})
+export class AppComponent {
+  /** Title of the application */
+  public title = 'tutorial-app';
+
+  /* Inject the ApiFactoryService and get the corresponding APIs */
+  private petApi;
+  private storeApi;
+
+  private readonly petsWritable = signal<Pet[]>([]);
+  public readonly pets = this.petsWritable.asReadonly();
+
+  private readonly petsInventoryWritable = signal<{ [key: string]: number }>({});
+  public readonly petsInventory = this.petsInventoryWritable.asReadonly();
```

Comment:
Shouldn't we use it in the template to ensure we receive data correctly ?

Reply:
updated